src/vcs/git.rs

```diff
@@ -788,6 +789,59 @@ impl<'a> Browser<'a> {
         self.repository.revision_branches(&commit.id())
     }
 
+    /// Get the [`Stats`] of the underlying [`Repository`].
+    ///
+    /// # Errors
+    ///
+    /// * [`error::Error::Git`]
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use radicle_surf::vcs::git::{Browser, Repository};
+    /// # use std::error::Error;
+    ///
+    /// # fn main() -> Result<(), Box<dyn Error>> {
+    /// let repo = Repository::new("./data/git-platinum")?;
+    /// let mut browser = Browser::new(&repo)?;
+    ///
+    /// let stats = browser.get_stats()?;
+    ///
+    /// // there should be 2 local branches
+    /// assert_eq!(stats.branch_count, 2);
+    ///
+    /// // there should be 14 commits
+    /// assert_eq!(stats.commit_count, 14);
+    ///
+    /// // 4 contributors: Alexander Simmerl, Fintan Halpenny, FintanH, Rudolfs Ošiņš
+    /// assert_eq!(stats.contributor_count, 4);
+    ///
+    /// # Ok(())
+    /// # }
+    /// ```
+    pub fn get_stats(&self) -> Result<Stats, Error> {
+        let history = self.repository.head()?;
```

Comment:
This shouldn't be needed

Reply:
The `?` or the whole line?

Reply:
The whole line. Sorry, I should have been clearer. The user will already be on a branch they're browsing so there's no need to set the history to head.